.github/workflows/main.yml

```diff
@@ -110,3 +110,21 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: brpaz/hadolint-action@master
+
+  deploy:
+    needs: [test, lint-dockerfile]
+    runs-on: ubuntu-latest
+
+    if: github.ref == 'refs/heads/master'
```

Comment:
or `main`?!

Reply:
When we switch the default branch on this repo I'll change this too!